From e522ff017db66756b4a67beb5efab2a6384d2c26 Mon Sep 17 00:00:00 2001
From: Radostin Stoyanov <rstoyanov@fedoraproject.org>
Date: Sun, 8 May 2022 10:53:34 +0100
Subject: [PATCH] makefile: Create PLUGINDIR only with CONFIG_AMDGPU

Building .deb package fails with:

mkdir: cannot create directory '/var/lib/criu': Permission denied

Signed-off-by: Radostin Stoyanov <rstoyanov@fedoraproject.org>
---
 plugins/amdgpu/Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/plugins/amdgpu/Makefile b/plugins/amdgpu/Makefile
index 84b9f8714..a49b4b04e 100644
--- a/plugins/amdgpu/Makefile
+++ b/plugins/amdgpu/Makefile
@@ -50,8 +50,8 @@ clean: amdgpu_plugin_clean amdgpu_plugin_test_clean
 mrproper: clean
 
 install:
-	$(Q) mkdir -p $(PLUGINDIR)
 ifeq ($(CONFIG_AMDGPU),y)
+	$(Q) mkdir -p $(PLUGINDIR)
 	$(E) "  INSTALL " $(PLUGIN_NAME)
 	$(Q) install -m 644 $(PLUGIN_SOBJ) $(PLUGINDIR)
 endif
-- 
2.35.3

